frontend/src/views/MovieAddScreen.vue

```diff
@@ -121,7 +121,7 @@ export default {
         if (this.image) {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`/api/cinema/movies-${movie.id}-upload-image/`, data, imageConfig);
         }
```

Comment:
you have to add `${import.meta.env.VITE_API_URL}` before `/api` and change `movies-${movie.id}-upload` to `movies/${movie.id}/upload`, otherwise, it won't work.